fork.html

```diff
@@ -0,0 +1,189 @@
+            <li><strong>Release process</strong>. Put in place a process for creating OpenTF releases.</li>  
+            <li><strong>Alpha release</strong>. Once the above items are done, we will create the first OpenTF release. This will be an alpha release, meant for testing by the community.</li>  
+          </ul>
+          <h4 class="small-margin">What's coming soon</h4>          
+          <ul class="tight-list emoji-list coming-soon-list">
+            <li><strong>Create an official OpenTF Registry via an RFC process</strong>. Go through an RFC process to create the official OpenTF Registry solution (replacing the initial solution).</li>  
+            <li><strong>Stable release</strong>. Create the first stable OpenTF release. This is meant for production usage, as a drop in replacement for Terraform, so we'll only do this release after sufficient testing and community feedback.</li>  
+          </ul>
+          <h4 class="small-margin">Join the OpenTF community!</h4>          
+          <p class="subtle-margin">
+            The response from the community so far has been incredible. In just a few weeks, more than 130 companies 
+            and 680 individuals have pledged support to the <a href="https://opentf.org/">OpenTF manifesto</a>, and the 
+            <a href="https://github.com/opentffoundation/manifesto">OpenTF manifesto repo</a> has gotten more than 
+            33,000 stars! By comparison, the Terraform repo took nearly 10 years to reach 38,000 stars:
+          </p>  
+          <img src="/images/star-history-202395.png" alt="OpenTF has gotten over 33,000 stars in GitHub in just a few weeks">
+          <p class="subtle-margin">
+            This sort of growth is unprecedent, and we're humbled by all of your support. As per the roadmap in the 
+            previous section, we're working hard on getting OpenTF to the point where we can start doing official 
+            releases. 
+          </p>
+          <p class="subtle-margin">
+            In the meantime, you can follow our progress at
+            <a href="https://github.com/opentffoundation/opentf">github.com/opentffoundation/opentf</a>,
+            contribute to the project by following the 
+            <a href="https://github.com/opentffoundation/opentf/blob/main/CONTRIBUTING.md">contribution guidelines</a>,
+            and provide feedback in the
+            <a href="https://join.slack.com/t/slack-9uv6202/shared_invite/zt-22ifsm1t2-AF6cL0cOdzivP8E~4deDJA">OpenTF Community Slack</a>. We are
+            thrilled to be working with the whole community in making OpenTF a truly open, community-driven project!
+          </p>
+        </article>
+      </section>
+
+      <section>
+        <h2>FAQ</h2>
```

Comment:
```suggestion
        <h2 id="faq">FAQ</h2>
```

Add ID so we can anchor to this part.

Reply:
Added!